pkgs/top-level/php-packages.nix

```diff
@@ -380,6 +391,59 @@ let
     };
   };
 
+  pinba = if isPhp73 then pinba73 else pinba7;
+
+  pinba7 = assert !isPhp73; buildPecl rec {
+    version = "1.1.1";
+    pname = "pinba";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "tony2001";
+      repo = "pinba_extension";
+      rev = "225262582ed3fb6e8de6eb5ec5a8c666716385dc";
```

Comment:
`rev = "RELEASE_1_1_1";` should do the same as this commit id. And be more clear that it matches the version number above.